spec/fixtures/api_responses/subject_areas.json

```diff
@@ -0,0 +1 @@
ject_knowledge_enhancement_course_available":null}},{"id":"31","type":"subjects","attributes":{"subject_name":"Religious education","subject_code":"V6","bursary_amount":"9000","early_career_payments":null,"scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"32","type":"subjects","attributes":{"subject_name":"Social sciences","subject_code":"14","bursary_amount":null,"early_career_payments":null,"scholarship":null,"subject_knowledge_enhancement_course_available":null}},{"id":"33","type":"subjects","attributes":{"subject_name":"Modern Languages","subject_code":null,"bursary_amount":null,"early_career_payments":null,"scholarship":null,"subject_knowledge_enhancement_course_available":null}},{"id":"34","type":"subjects","attributes":{"subject_name":"French","subject_code":"15","bursary_amount":"26000","early_career_payments":"2000","scholarship":"28000","subject_knowledge_enhancement_course_available":true}},{"id":"35","type":"subjects","attributes":{"subject_name":"English as a second or other language","subject_code":"16","bursary_amount":null,"early_career_payments":null,"scholarship":null,"subject_knowledge_enhancement_course_available":null}},{"id":"36","type":"subjects","attributes":{"subject_name":"German","subject_code":"17","bursary_amount":"26000","early_career_payments":"2000","scholarship":"28000","subject_knowledge_enhancement_course_available":true}},{"id":"37","type":"subjects","attributes":{"subject_name":"Italian","subject_code":"18","bursary_amount":"26000","early_career_payments":"2000","scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"38","type":"subjects","attributes":{"subject_name":"Japanese","subject_code":"19","bursary_amount":"26000","early_career_payments":"2000","scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"39","type":"subjects","attributes":{"subject_name":"Mandarin","subject_code":"20","bursary_amount":"26000","early_career_payments":"2000","scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"40","type":"subjects","attributes":{"subject_name":"Russian","subject_code":"21","bursary_amount":"26000","early_career_payments":"2000","scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"41","type":"subjects","attributes":{"subject_name":"Spanish","subject_code":"22","bursary_amount":"26000","early_career_payments":"2000","scholarship":"28000","subject_knowledge_enhancement_course_available":true}},{"id":"42","type":"subjects","attributes":{"subject_name":"Modern languages (other)","subject_code":"24","bursary_amount":"26000","early_career_payments":"2000","scholarship":null,"subject_knowledge_enhancement_course_available":true}},{"id":"43","type":"subjects","attributes":{"subject_name":"Further education","subject_code":"41","bursary_amount":null,"early_career_payments":null,"scholarship":null,"subject_knowledge_enhancement_course_available":null}}],"jsonapi":{"version":"1.0"}}
```

Comment:
Can your IDE easily format the JSON like so? https://github.com/DFE-Digital/find-teacher-training/blob/master/spec/fixtures/api_responses/courses.json

- Makes a bit more readable but not a deal breaker

Reply:
🤣 My 'IDE' can barely even open that file. I've tried formatting it but there must be an error in it somewhere when I do as it starts failing loads of tests with a weird seemingly unrelated error message. I'm going to merge and I'll sort that out in a different PR.